### ISSUE TITLE
 Anonymous interfaces embedded in other interfaces should be type-checked

### ISSUE THREAD

**Issue by zakjan** [MeirionHughes/aurelia-template-lint#172]
item.ts
```
export interface Item {
    attributes: {
        name: string;
    };
}
```

test.ts
```
import {Item} from "./item";

export class TestViewModel {
    item: Item;
}
```

test.html
```
<template>
    ${item.attributes.aaa}
</template>
```

This should throw a warning about missing attribute aaa, but it is not detected.

---

Note that if I extract attributes object into a separate interface, it starts working.

item.ts
```
export interface ItemAttributes {
    name: string;
}

export interface Item {
    attributes: ItemAttributes;
}
```

cannot find 'aaa' in type 'ItemAttributes' Ln 3 Col 5 /src/demo/vm/data/test.html

In this case, I think that message "cannot find 'attributes.aaa' in type 'Item'" would be more helpful, because anyway that's the same message that will probably be displayed in the first case.

**Comment by zakjan** [comment]
Duplicate of #65, closing.
